app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,134 @@
  *= require_tree .
  *= require_self
  */
+
+ body {
+    font-family: "Lato", sans-serif;
+    color: #3b8070;
+    text-align: center;
+    background-color: rgb(230, 235, 236);
+ }
+
+ input[type=submit], button {
+    display: inline-block;
+    font-family: "Roboto", sans-serif;
+    font-weight: bold;
+    font-size: 18px;
```

Comment:
Non mi piace molto il font-size statico in px qui, in generale i px statici andrebbero usati poco. Se riesci a fare una soluzione più elegante in poco tempo ok facciamola, sennò lascia così e ci penseremo